webapp/src/main/java/com/box/l10n/mojito/android/strings/AndroidStringDocument.java

```diff
@@ -37,7 +38,7 @@ public void addSingular(AndroidStringElement element, Node comment) {
         addSingular(new AndroidSingular(element.getIdAttribute(),
                 element.getNameAttribute(),
                 element.getUnescapedContent(),
-                comment.getTextContent()));
+                Optional.ofNullable(comment).map(Node::getTextContent).orElse(null)));
```

Comment:
what's the point of creating an optional here vs just using the simpler ternary operator?

Reply:
updated